element/index.js

```diff
@@ -65,3 +66,20 @@ export function renderToString( element ) {
 
 	return renderToStaticMarkup( element );
 }
+
+export function concatValues( value1, value2 ) {
```

Comment:
Is this function effectively concatenating two element trees? Could the name more accurately reflect this? "values" is not a term that means much to me in a React sense.